workspace-manager-clienttests/src/main/resources/configs/integration/WorkspaceLifecycle.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "CreateGetDeleteWorkspace",
```

Comment:
A couple extra changes to make with this:

- This file is the integration test case, but we have a similar test for performance with different parameters at workspace-manager-clienttests/src/main/resources/configs/perf/CreateGetDeleteWorkspace.json. could you update that as well?
- This file (and the perf test) is referenced by a test suite, please update the name there as well (workspace-manager-clienttests/src/main/resources/suites/BasicIntegration.json and BasicPerf.json)

Reply:
Done (I think)